### ISSUE TITLE
Rename “Skill Code” to  “Skill Cone (Code)”

### ISSUE THREAD

**Issue by burgwyn** [18F/State-TalentMAP#1790]
Rename “Skill Code” to  “Skill Cone (Code)”

**Comment by burgwyn** [comment]
I'm sure about the spacing on this card.  Do we want to trim the label to "Skill code" in this case?
<img width="401" alt="screen shot 2018-04-19 at 8 36 06 pm" src="https://user-images.githubusercontent.com/661204/39024838-626e3a24-4411-11e8-8b1b-1b41a9e47338.png">

**Comment by mjoyce91** [comment]
Here's a few different instances of how it would look.

<img width="319" alt="screen shot 2018-04-20 at 9 32 43 am" src="https://user-images.githubusercontent.com/11576347/39053815-f32f0fea-447d-11e8-9329-04a37feb5648.png">
<img width="896" alt="screen shot 2018-04-20 at 9 32 49 am" src="https://user-images.githubusercontent.com/11576347/39053816-f33cdd00-447d-11e8-86b5-8f8cd116ced0.png">
<img width="689" alt="screen shot 2018-04-20 at 9 32 56 am" src="https://user-images.githubusercontent.com/11576347/39053817-f347e39e-447d-11e8-8243-7abbfba74c10.png">
<img width="637" alt="screen shot 2018-04-20 at 9 33 09 am" src="https://user-images.githubusercontent.com/11576347/39053818-f355c388-447d-11e8-8135-8d921c79ccb3.png">
<img width="268" alt="screen shot 2018-04-20 at 9 33 28 am" src="https://user-images.githubusercontent.com/11576347/39053819-f35f4430-447d-11e8-9dd3-eca8f333974a.png">

**Comment by cydneycappello** [comment]
@mjoyce91 I agree with you that Cone is ONLY shown in the filters, otherwise it's the Skill Code bidders are seeing on the homepage cards, search results cards, and position detail page. So I agree we should only show the word Cone on the filters.

We may want to just let Chris and Robin know we're thinking of this change, but I do believe this suggestions would be much clearer for bidders.
